uarray/_uarray_dispatch.cxx

```diff
@@ -889,9 +889,11 @@ int Function::clear(Function * self)
 }
 
 
+// getset takes mutable char * in python < 3.7
+static char dict__[] = "__dict__";
```

Comment:
Is it possible to squeeze a const somewhere in here?

Reply:
If the buffer was const then the pointer would have to be `const char *` and it wouldn't work.